schema.graphql

```diff
@@ -253,10 +253,8 @@ type FpmmTransaction @entity {
   user: Account!
   fpmmType: FpmmType!
   transactionType: TransactionType!
-  collateralAmount: BigInt!
-  collateralAmountUSD: BigDecimal!
-  collateralTokenAddress: Bytes!
   collateralTokenAmount: BigInt!
+  sharesOrPoolTokenAmount: BigInt!
```

Comment:
This is an interesting choice. Maybe it'd be worthwhile to split this into two fields, though that is certainly not as optimized as this in terms of space, and if you were to split this into two fields, then it would be right to make both fields optional as well.

So yeah, I guess this should stay as is.